function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
```

Comment:
```suggestion
def factorial(n)
```
Maybe this method could use a more descriptive name